app/root.tsx

```diff
@@ -1,39 +1,28 @@
+import { useSWEffect } from '~/utils/client/sw-hook';
+import type { V2_MetaFunction } from '@remix-run/react';
 import {
   Links,
   LiveReload,
   Meta,
   Outlet,
   Scripts,
   ScrollRestoration,
-  V2_MetaFunction,
 } from '@remix-run/react';
-import { LinksFunction, LoaderFunction, json, redirect } from '@remix-run/node'; // or cloudflare/deno
+import type { LinksFunction } from '@remix-run/node'; // or cloudflare/deno
 import styles from './tailwind.css';
-import { AUTHENTICATED } from './models/app.constants';
-
 export const links: LinksFunction = () => [{ rel: 'stylesheet', href: styles }];
 export const meta: V2_MetaFunction = () => {
-  return [
-    { title: 'Ore+' },
-    {
-      charSet: 'utf8',
-    },
-  ];
-}; //TODO: charSet Does not seem to be working globally
```

Comment:
```
  return [
    { title: 'Orem+' },
    // eslint-disable-next-line unicorn/text-encoding-identifier-case
    { charSet: 'utf-8' },
    {
      name: 'viewport',
      content: 'width=device-width,initial-scale=1'
    }
  ];
  ```

Reply:
I have found an article explaining that I should be using meta tags above the Meta component if they ment to be global otherwise they will be gone if I ever call a nested meta function